README.md

```diff
@@ -73,9 +73,37 @@ Add the `expo.hooks` object to your project's `app.json` (or `app.config`) file:
 }
 ```
 
+> If you're using the bare workflow, or using a plain React Native app, you should also follow [these quick steps](#bare-workflow-setup) to finish up your setup
+
 All done! For more information on customization and additional features, read the full guide on using Sentry in Expo apps in our docs ➡️ ["Using
 Sentry"](https://docs.expo.io/guides/using-sentry/).
 
+### Bare workflow setup
+
+Setting up `sentry-expo` in the bare workflow requires just a few extra steps in addition to those listed above:
+
+1. Run `yarn sentry-wizard -i reactNative -p ios android`. This will automatically configure your native Android & iOS projects
+2. The previous step will add an extra
+
+   ```
+   import * as Sentry from '@sentry/react-native';
+
+   Sentry.init({
+     dsn: 'YOUR DSN',
+   });
+   ```
+
+to your root project file (usually `App.js`), so make sure you remove it (but keep the `sentry-expo` import and original `Sentry.init` call!)
+
+If you don't rely on `expo-updates`, you're all done! If you do, read on...
```

Comment:
Maybe a sub-header is a bit easier for people just scanning the docs? e.g.

```suggestion
#### Configure `expo-updates`

> Skip this step if you don't rely on or use `expo-updates`
```

Reply:
good call, I'm actually going to label it `Configure "no publish builds"` since that's what they really are. I know people don't automatically know what no publish builds are, but the next paragraph will explain that